documentation/projects/proposals/additional_search_views/20230424-project_proposal_additional_search_views.md

```diff
@@ -0,0 +1,93 @@
+# 2023-04-24 Project Proposal: Additional search views
+
+**Author**: @fcoveram
+
+## Reviewers
+
+- [x] @obulat
+- [ ] @krysal
+
+## Project summary
+
+The Openverse team has decided to enhance the browsing experience by
+incorporating new content views for providers, tags, and creators. This feature
+will allow users to share collections detached of individual searches.
+
+## Goals
+
+Search Experience
+
+## Requirements
+
+The project adds three additional views for the Nuxt frontend where users
+interact in the following ways.
+
+### Results by tag
+
+Users land on the tag results page by clicking on the tag in the single result
+page or by opening a tag results link (for instance,
+[www.openverse.org/tags/image/cat](https://www.openverse.org/tags/image/cat)).
+
+The results page shows the name of the tag and all the items of the selected
+media type that have this exact tag applied. The items are sorted by
+`ingested_on` field in descending order.
+
+For instance, the `cat` tag page will show only images that have the `cat` tag,
+not items tagged with `cats` .
```

Comment:
```suggestion
For instance, the `cat` tag page will show only images that have the `cat` tag,
not items tagged with `cats`.
```
Wouldn't `cats` be also relevant for the `cat` page and vice-versa? It would be a bit of a pain to have to add both tags. CC @obulat @AetherUnbound

Reply:
I think these would be created dynamically based on the tags which exist for a result, we wouldn't be manually adding any of the tags to existing media. This would make the tags serve as a strict match on other tags, if I'm understanding it correctly.

Reply:
I will resolve this conversation to continue with the Implementation Plan(s), and in case we include `cats` within the results, I'll go back to this document to update it.